src/scene/vcFlythrough.cpp

```diff
@@ -269,6 +269,9 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
   switch (m_state)
   {
   case vcFTS_None:
+    if (m_flightPoints.length > 2 && ImGui::Button(vcString::Get("flythroughSmooth")))
```

Comment:
Can you make this fit in with the layout changes?

Reply:
when u say fit in with the layout changes, do you mean just add {}'s?